### PR TITLE
Rds/cml/function are all hvac sys fan objs autosized

### PR DESCRIPTION
**Pull request by claperle**

### CODE REVIEW

File: docs/ruleset_functions/are_all_hvac_sys_fan_objs_autosized.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+- **are_all_hvac_sys_fan_objs_autosized**: Returns true or false. The function returns true if all fans objects associated with an hvac system are autosized.   
+ 
+**Function Call:** 
+1. get_dict_of_zones_and_terminal_units_served_by_hvac_sys()    
+
+## Logic:  
+- Create dictionary of hvac systems and associated zones and terminal units: `dict_of_zones_and_terminal_units_served_by_hvac_sys_x = get_dict_of_zones_and_terminal_units_served_by_hvac_sys(RMI)`  
+- Set are_all_hvac_sys_fan_objs_autosized = true: `are_all_hvac_sys_fan_objs_autosized = true`  
+- Check if there is a fan system associated with the hvac system (if not then assume it is defined at the terminal unit like it is for a four-pipe fan coil unit): `if hvac.fan_system != Null:`
+    - For each supply fan associated with the fan system: `for supp_fan in hvac.fan_system.supply_fans:`
```
Comment (mkarpman) [inline docs/ruleset_functions/are_all_hvac_sys_fan_objs_autosized.md:19]: I think we should only check that supply fans are autosized.

File: docs/ruleset_functions/are_all_hvac_sys_fan_objs_autosized.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+- **hvac_sys_obj**: The hvac object in which the associated fan objects will be assessed.
+
+**Returns:**  
+- **are_all_hvac_sys_fan_objs_autosized**: Returns true or false. The function returns true if all fans objects associated with an hvac system are autosized.   
+ 
+**Function Call:** 
+1. get_dict_of_zones_and_terminal_units_served_by_hvac_sys()    
+
+## Logic:  
+- Create dictionary of hvac systems and associated zones and terminal units: `dict_of_zones_and_terminal_units_served_by_hvac_sys_x = get_dict_of_zones_and_terminal_units_served_by_hvac_sys(RMI)`
```
Comment (mkarpman) [inline docs/ruleset_functions/are_all_hvac_sys_fan_objs_autosized.md:16]: This is just for one system, right? So the logic description should be to create dictonary of zones and terminal units associated with the HVAC system, correct?

Comment (claperle) [reply docs/ruleset_functions/are_all_hvac_sys_fan_objs_autosized.md:16]: @mkarpman This dictionary has keys for multiple hvac systems. We extract the information for the hvac we care about at line 28 so I think the logic is correct.

File: docs/ruleset_functions/are_all_hvac_sys_fan_objs_autosized.md (hunk 1 above)
Comment (claperle) [reply docs/ruleset_functions/are_all_hvac_sys_fan_objs_autosized.md:19]: Updated accordingly

**Review by supriyagoel — APPROVED** [review]

**Review by mkarpman — APPROVED** [review]

**Pull request merged**